welltestpy/tools/diagnostic_plots.py

```diff
@@ -90,10 +90,8 @@ def diagnostic_plot_pump_test(
         if keep_fs:
             plt.rcParams.update({"font.size": font_size})
         fig, ax = plotter._get_fig_ax(fig, ax)
-        ax.scatter(x, y, color="red", label="drawdown")
-        ax.plot(sx, sy, c="red")
-        ax.plot(dx, dy, c="black", linestyle="dashed")
-        ax.scatter(dx, dy, c="black", marker="+", label="time derivative")
+        ax.scatter(x, y, color="blue", label="drawdown")
+        ax.plot(dx, dy, c="red", label="time derivative")
```

Comment:
How does this look now? Can you provided an example?

Reply:
I wanted to make it look like your plots for the diagnostic plots with the data only in blue dots and the derivative in smooth red.

Reply:
![Figure_1](https://user-images.githubusercontent.com/82570263/128836385-922dc7f4-bc13-40fe-875d-73f5c82aff44.jpeg)